src/rocqproverorg_frontend/pages/install.eml

```diff
@@ -222,75 +271,100 @@ hybrid Rocq/markdown document support.
-            <a href="https://github.com/coq-community/vscoq-legacy/">VsCoq Legacy</a> is a legacy Rocq extension for Visual Studio Code that only relies on the RocqIDE protocol and therefore does not require installing a language server. You can install VsCoq Legacy from the <a href="https://marketplace.visualstudio.com/items?itemName=coq-community.vscoq1">Visual Studio Code Marketplace</a> or from <a href="https://open-vsx.org/extension/coq-community/vscoq1">Open VSX</a>.
-          </li>
-        </ul>
+        </p>
+
         <h3>
-          Emacs
+        VsCoq Legacy
+        </h3>
+        <p>
+          <a href="https://github.com/coq-community/vscoq-legacy/">VsCoq Legacy</a> is a legacy Rocq extension for Visual Studio Code that only relies on the RocqIDE protocol and therefore does not require installing a language server. You can install VsCoq Legacy from the <a href="https://marketplace.visualstudio.com/items?itemName=coq-community.vscoq1">Visual Studio Code Marketplace</a> or from <a href="https://open-vsx.org/extension/coq-community/vscoq1">Open VSX</a>.
+        </p>
+      </div>
+
+      <div class="prose dark:prose-invert" x-show="(editor == 'emacs')">
+        <h2>
+          Set Up Emacs for Rocq
+        </h2>
+        <p>
+          After installing the Rocq Prover, you should set up Emacs
+          to write Rocq code and proofs.
+        </p>
+        <h3>
+          Proof General
         </h3>
         <p>
           Emacs users usually use <a href="https://proofgeneral.github.io/">Proof General</a> to interact with the Rocq Prover.
           It can be installed from the <a href="https://melpa.org/#/proof-general">MELPA repository</a>.
           It is often complemented with <a href="https://github.com/cpitclaudel/company-coq">Company-Coq</a> minor mode, which can also be installed from the MELPA repository.
         </p>
-        <h3>
-          Vim / Neovim
-        </h3>
+      </div>
+
+      <div class="prose dark:prose-invert" x-show="(editor == 'vim')">
+        <h2>
+          Set Up VS Vim / Neovim for Rocq
+        </h2>
+        <p>
+          After installing the Rocq Prover, you should set up Vim / Neovin to
+          write Rocq code and proofs.
+        </p>
         <p>
           Vim users usually use <a href="https://github.com/whonore/Coqtail">Coqtail</a> to interact with the Rocq Prover. Various installation methods are documented in the repository's README.
         </p>
-        <h3>
-          Standalone IDE
-        </h3>
+      </div>
+
+      <div class="prose dark:prose-invert" x-show="(editor == 'rocqide')">
+        <h2>
+          Set Up Rocq IDE
+        </h2>
         <p>
-          The Rocq Prover also has a standalone IDE called RocqIDE. It is distributed as part of the Rocq Platform.
+          The Rocq Prover also has a standalone IDE called RocqIDE. It is distributed as part of the Rocq Platform and is available in many Linux distributions (sometimes as a separate package).

```

Comment:
This appears to have been copied from the Linux section, but the "and is available in many Linux distributions" part is not relevant for macOS.